src/test/java/uk/nhs/hee/tis/trainee/forms/api/LtftResourceTest.java

```diff
@@ -86,4 +93,81 @@ void shouldGetLtftSummariesWhenLtftFormsExist() {
     assertThat("Unexpected ID.", responseDto2.id(), is(id2));
     assertThat("Unexpected name.", responseDto2.name(), is("Test LTFT 2"));
   }
+
+  @Test
+  void shouldReturnBadRequestWhenServiceWontSaveLtftForm() {
+    when(service.saveLtftForm(any())).thenReturn(Optional.empty());
+
+    ResponseEntity<LtftFormDto> response = controller.createLtft(new LtftFormDto());
+
+    assertThat("Unexpected response code.", response.getStatusCode(), is(BAD_REQUEST));
+  }
+
+  @Test
+  void shouldReturnBadRequestWhenServiceWontUpdateLtftForm() {
+    when(service.updateLtftForm(any(), any())).thenReturn(Optional.empty());
+
+    ResponseEntity<LtftFormDto> response = controller.updateLtft(ID, new LtftFormDto());
+
+    assertThat("Unexpected response code.", response.getStatusCode(), is(BAD_REQUEST));
+  }
+
+  @Test
+  void shouldReturnNotFoundWhenServiceCantFindLtftForm() {
+    when(service.getLtftForm(any())).thenReturn(Optional.empty());
+
+    ResponseEntity<LtftFormDto> response = controller.getLtft(ID);
+
+    assertThat("Unexpected response code.", response.getStatusCode(), is(NOT_FOUND));
+  }
+
+  @Test
+  void shouldReturnSavedLtftFormWhenSaved() {
+    LtftFormDto savedForm = new LtftFormDto();
+    savedForm.setId(ID);
+    savedForm.setTraineeTisId("some trainee");
+    when(service.saveLtftForm(any())).thenReturn(Optional.of(savedForm));
+
+    LtftFormDto newForm = new LtftFormDto();
+    newForm.setTraineeTisId("some trainee");
+    ResponseEntity<LtftFormDto> response = controller.createLtft(newForm);
+
+    assertThat("Unexpected response code.", response.getStatusCode(), is(OK));
+    LtftFormDto responseDto = response.getBody();
+    assert responseDto != null;
+    assertThat("Unexpected response body.", responseDto.equals(savedForm), is(true));
```

Comment:
Is there a reason not to use `assertThat("", responseDto, is(savedForm))`?
As-is, failure messages are going to be pretty poor.